src/main/drivers/motor.c

```diff
@@ -60,15 +60,49 @@ void motorWriteAll(float *values)
 {
 #ifdef USE_PWM_OUTPUT
     if (motorDevice->enabled) {
+#ifdef USE_DSHOT_BITBANG
+        if (isDshotBitbangActive(&motorConfig()->dev)) {
+            // Initialise the output buffers
+            if (motorDevice->vTable.updateInit) {
+                motorDevice->vTable.updateInit();
+            }
+
+            // Update the motor data
+            for (int i = 0; i < motorDevice->count; i++) {
+                motorDevice->vTable.write(i, values[i]);
+            }
+
+            // Don't attempt to write commands to the motors if telemetry is still being received
+            if (motorDevice->vTable.telemetryWait) {
```

Comment:
Consistent with what? You mean I should compare against NULL or add a `Null` version of this routine as a default? Quicker not to call it I suggest.

Reply:
Most of virtual function in `motorDevice.vTable` use empty (`*Null`) function instead of `NULL` pointer. So `telemetryWait` can be implemented in similar manner, saving `NULL` test and adding cycle or two of overhead (only in `NULL` case).
Main gain is code consistency, influence on performance shall be negligible.